python/triton/language/semantic.py

```diff
@@ -58,14 +58,21 @@ def computation_type_impl(a_ty: tl.dtype, b_ty: tl.dtype, div_or_mod: bool) -> t
         return tl.float32
     # 3 ) if one operand is half, the other is implicitly converted to half
     #     unless we're doing / or %, which do not exist natively in PTX for fp16.
+    #     Supported PTX op: add, sub, mul, fma, neg, abs, min, max, tanh, ex2, setp
```

Comment:
It feels weird to mention PTX and supported ops here. This function is just meant to implement implicit casting rules, and these are valid for any binary operator that perform implicit conversions.